metron-platform/metron-common/src/main/java/org/apache/metron/common/Constants.java

```diff
@@ -43,6 +43,7 @@
     ,DST_PORT("ip_dst_port")
     ,PROTOCOL("protocol")
     ,TIMESTAMP("timestamp")
+    ,ORIGINAL("original_string")
```

Comment:
I particularly like this one.  We should refactor the GrokParser to use it as a follow-on.